cmake/ExtraRepositoriesList.cmake

```diff
@@ -0,0 +1,10 @@
+SET_DEFAULT_AND_FROM_ENV(Futility_URL_REPO_BASE
+  $ENV{USER}@ners-arc-05.engin.umich.edu:/git-root/MPACT/)
```

Comment:
you have to have a base location to clone the repos if we want clone extra repositories to work.  Unfortunately, TeuchosWrappers isn't open yet.